es2022.md

```diff
@@ -83,6 +83,33 @@ extend interface BinaryExpression <: Expression {
 - `left` can be a private identifier (e.g. `#foo`) when `operator` is `"in"`.
 - See [Ergonomic brand checks for Private Fields][proposal-private-fields-in-in] for details.
 
+# Modules
+
+See https://github.com/tc39/ecma262/pull/2154 for more details.
```

Comment:
Nit: Consider provide a text link instead of url, for example, `[Arbitrary module namespace identifier names]`.